### PR TITLE
[SECD-441] Add init repo updates and add initial interfaces

### PR DESCRIPTION
**Pull request by austburn**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
-FROM golang:latest AS BUILDER
-WORKDIR $GOPATH/src/github.com/asecurityteam/nexpose-vuln-hydrator
-COPY . .
+FROM asecurityteam/sdcli:v1 AS BUILDER
+RUN mkdir -p /go/src/github.com/asecurityteam/nexpose-vuln-filter
```
Comment (bkrebsbach) [inline Dockerfile]: Task: Should be `/go/src/github.com/asecurityteam/nexpose-vuln-hydrator`

File: api.yaml [hunk 2]
```diff
@@ -46,31 +46,21 @@ info:
     name: Apache 2.0
     url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
 paths:
-  /healthcheck:
```
Comment (bkrebsbach) [inline api.yaml]: Task: The healthcheck endpoint should be a part of the spec.

File: api.yaml [hunk 3]
```diff
@@ -46,31 +46,21 @@ info:
...
-      description: An API Call.
-      parameters:
-        - name: "name"
-          in: "path"
-          description: "Name of person to greet."
-          required: true
-          schema:
-            type: "string"
+  /hydrate:
+    postt:
```
Comment (bkrebsbach) [inline api.yaml]: Task: Typo `postt` -> `post`

File: api.yaml (hunk 2 above)
Comment (zlozano) [inline api.yaml]: question: even though this api.yaml isn't really functional, why get rid of the healthcheck?

File: pkg/handlers/v1/hydrator.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	Port     int32  `json:"port"`
+	Protocol string `json:"protocol"`
+}
+
+// Handle accepts AssetEvents and returns a hydrated asset containing vulnerability information
+func (h *HydrationHandler) Handle(ctx context.Context, evt AssetEvent) (AssetVulnerabilitiesEvent, error) {
+	asset := domain.Asset(evt)
+	assetWithVulnerabilityDetails, err := h.Hydrator.HydrateVulnerabilities(asset)
+	if err != nil {
+		return AssetVulnerabilitiesEvent{}, err
```
Comment (zlozano) [inline pkg/handlers/v1/hydrator.go]: question: would an error log be suitable here?

File: api.yaml (hunk 2 above)
Comment (austburn) [reply api.yaml]: added it back

File: pkg/handlers/v1/hydrator.go (hunk 4 above)
Comment (austburn) [reply pkg/handlers/v1/hydrator.go]: I can add that - a good idea to get some of the mocking boilerplate in for future PRs

**Pull request merged**